make.go

```diff
@@ -689,7 +690,7 @@ func writeITP(gopkg, debsrc, debversion string) (string, error) {
 		description = "TODO"
 	}
 
-	fmt.Fprintf(f, "From: %q <%s>\n", getDebianName(), getDebianEmail())
+	fmt.Fprintf(f, "From: %q <%s>\n", mime.QEncoding.Encode("utf-8", getDebianName()), getDebianEmail())
 	fmt.Fprintf(f, "To: submit@bugs.debian.org\n")
 	fmt.Fprintf(f, "Subject: ITP: %s -- %s\n", debsrc, description)
```

Comment:
This could be impacted as well?

Reply:
Fixed in 9374c6744f64831fbd14601c1cc6d8e686b61060.